tests/eclient/testdata/dev_local_info.txt

```diff
@@ -0,0 +1,241 @@
+eden controller edge-node update --device local_profile_server=$app_ip:8888
+
+# STEP 1: Wait for devinfo status
+exec -t 1m bash get-devinfo-status.sh
+stdout 'ONLINE'
+! stderr .
+
+# STEP 2: Deploy the second app
+eden pod deploy -n app1 --memory=512MB docker://lfedge/eden-eclient:9455582 -p {{template "app_port"}}:22 --networks={{template "network"}}
+test eden.app.test -test.v -timewait 10m RUNNING app1
+
+# Wait for ssh access
+exec -t 5m bash wait-ssh.sh {{template "app_port"}}
+
+# STEP 3: Wait for new devinfo status
+exec sleep 30
+exec -t 1m bash get-devinfo-status.sh
+stdout 'ONLINE'
+! stderr .
+
+# STEP 4: Request for shutdown. Check that app1 stops before local_manager
+exec -t 1m bash put-devinfo-cmd.sh COMMAND_SHUTDOWN &
+exec -t 5m bash wait-for-dev-state.sh SHUTTING_DOWN
+! stderr .
+# Could it already have HALTED? Match HALTING or HALTED
+exec -t 5m bash wait-for-app-state.sh app1 HALT
+! stderr .
+exec -t 1m bash get-appinfo-status.sh local-manager
+! stdout 'RUNNING'
+! stderr .
+
+# Could it already have HALTED? Match HALTING or HALTED
+exec -t 5m bash wait-for-app-state.sh local-manager HALT
+! stderr .
+exec -t 1m bash get-appinfo-status.sh app1
+! stdout 'HALTED'
+! stderr .
+
+# STEP 5: test reboot via controller bringing them back up
+# send reboot command and wait in background
+test eden.reboot.test -test.v -timewait=20m -reboot=1 -count=1 &
+exec -t 5m bash wait-for-app-state.sh local-manager RUNNING
+! stderr .
+exec -t 5m bash wait-for-app-state.sh app1 RUNNING
+! stderr .
+
+# STEP 6: Request for poweroff. Check that app1 stops before local_manager
+exec -t 1m bash put-devinfo-cmd.sh COMMAND_SHUTDOWN_POWEROFF &
+exec -t 5m bash wait-for-dev-state.sh POWERING_OFF
+! stderr .
+# Could it already have HALTED? Match HALTING or HALTED
+exec -t 5m bash wait-for-app-state.sh app1 HALT
+! stderr .
+exec -t 1m bash get-appinfo-status.sh local-manager
+! stdout 'RUNNING'
+! stderr .
+
+# Could it already have HALTED? Match HALTING or HALTED
+exec -t 5m bash wait-for-app-state.sh local-manager HALT
+! stderr .
+exec -t 1m bash get-appinfo-status.sh app1
+! stdout 'HALTED'
+! stderr .
+
+# STEP 7: Check qemu process is gone aka powered off
+exec sleep 120
+eden eve status
+! stdout process not running'
+
+# STEP 8: Restart EVE; check apps come up
+message 'Resetting of EVE'
+eden eve reset
+exec sleep 30
+
+# STEP 8.1: Wait for devinfo status
+exec -t 1m bash get-devinfo-status.sh
+stdout 'ONLINE'
+! stderr .
+
+# STEP 8.2: Wait for appinfo status
+exec -t 1m bash get-appinfo-status.sh
+stdout 'local-manager'
+stdout 'app1'
+! stderr .
+
+
+# STEP 9: Remove the second app
+eden pod delete app1
+test eden.app.test -test.v -timewait 15m - app1
+
+# STEP 10: Undeploy local-manager
+eden pod delete local-manager
+test eden.app.test -test.v -timewait 15m - local-manager
+eden network delete {{template "network"}}
+test eden.network.test -test.v -timewait 10m - {{template "network"}}
+
```

Comment:
We should put `eden eve reset` and small sleep here to remove configuration of LPS from EVE config.

Reply:
Before the pod delete? Or before the delete of the local-manager?

I don't think we have that step in the other uses of the local profile server; I copied this from app_local_info.txt

Reply:
Oh, seems I missed line here, after waiting for network delete, in the end.
Also please update eve version to one with PR merged here https://github.com/lf-edge/eden/blob/99d7f6244a1b98b3fb1b3372f0346bb44babebe1/pkg/defaults/defaults.go#L53

Reply:
When do we update DefaultEVETag? Any time there is a new feature in EVE-OS needed to pass the tests?

Reply:
Yes, we update version inside Eden repo when we have new feature in EVE test need to run. Inside EVE repo we use current (the last) version of EVE for tests.